.github/patch-initial-checklist.md

```diff
@@ -1,18 +1,24 @@
 The release pull request has been created! This checklist is a guide to follow for the remainder of the patch release process. You can check off each item in this list once completed.
 
-## Checklist
-
 * [ ] Checkout the release branch locally.
-* [ ] Create a GitHub milestone for the next release if necessary (set the start date as today's date). In some cases this may not be necessary for a patch release if there is already a minor release milestone present.
-* [ ] If there are any remaining open issues in the milestone for {{version}}, do a final check with the team to verify none of those *need* to go into the release. After verifying, go through and move any open issues/pulls into the next milestone (except for any that have the `type: blocker` label). After this any `type: blocker` labelled issues must be complete and merged into the release branch (cherry-pick if necessary) before continuing.
-* [ ] If this patch release is the latest release. Copy the changelog from the pull request description into the `readme.txt` file in the changelog section. Create a new section here for this release, eg.g. `= {{version}} - 2020-01-20 =`.
-* [ ] If this patch release is the latest release. Make any other changes to plugin metadata as necessary (no version changes needed, that's handled by script later in the process).
+
+## Initial Prep - changelog and meta data changes.
+
+The below only needs done if this patch release is the **_latest_** release of the plugin.
+
+* [ ] Copy the changelog from the pull request description into the `readme.txt` file in the changelog section. Create a new section here for this release, eg.g. `= {{version}} - 2020-01-20 =`.
```

Comment:
Might be good to specify if this section should be created at the top of the `== Changelog ==` or right above the related major/minor version. In other words, changelog should be ordered by release date or by version?

For example: if the last releases are 3.2 and 3.3 and we are releasing 3.2.1, how should changelog look:

Ordered by date:
```
= 3.2.1 - 2020-09-09 =
...
= 3.3.0 - 2020-09-02 =
...
= 3.2.0 - 2020-08-17 =
...
```

Ordered by version:
```
= 3.3.0 - 2020-09-02 =
...
= 3.2.1 - 2020-09-09 =
...
= 3.2.0 - 2020-08-17 =
...
```